hack/images.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
part of this or used for release?

Reply:
yep, I kind of added it by accident, but it makes sense to get it pushed, as I used it for couple of months